app/javascript/dashboard/helper/actionCable.js

```diff
@@ -43,6 +45,37 @@ class ActionCableConnector extends BaseActionCableConnector {
   onStatusChange = data => {
     this.app.$store.dispatch('updateConversation', data);
   };
+
+  onTypingOn = ({ conversation, user }) => {
+    this.clearTimer();
```

Comment:
1. This should not be in baseActionCableConnector
2. A timer should be attached to a conversationID